chatbot.py

```diff
@@ -17,35 +25,35 @@ def setup():
     logging.debug('Classification model ready')
     databaseconnect.setup_database()
     logging.debug('Database setup completed, database connected')
-    learn_response = 0
+    learn_response = LearnResponse.MESSAGE.name
     return clf, learn_response
 
 
 @logger_config.logger
 def message_to_bot(H, clf, learn_response):
-    if learn_response == 2:
+    if learn_response == LearnResponse.DESTINATION.name:
```

Comment:
@Nandini9634 @vishakha-lall correct me if I'm wrong, but isn't the earlier learn_response value 2 for origin?